Libraries/xcsdk/Headers/xcsdk/SDK/Manager.h

```diff
@@ -88,7 +89,7 @@ class Manager {
     /*
      * Load from a developer root. Returns nullptr on error.
      */
-    static std::shared_ptr<Manager> Open(libutil::Filesystem const *filesystem, std::string const &path);
+    static std::shared_ptr<Manager> Open(config::Config const *config, libutil::Filesystem const *filesystem, std::string const &path);
```

Comment:
It's a subtle distinction, and I'm not sure if it's common in C++ or just in this code. To me, a pointer "means" passing a resource while explicitly not copying it, where a const reference is just an optimization for passing by value.

Since a `Filesystem` is an expensive thing to copy (conceptually, copying the entire filesystem it represents would be ridiculous), a pointer makes sense. But especially if `Config` becomes a value type, there's no reason not to pas by value, and the const reference would just make that faster.